<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <widget class="QWidget" name="M3UCreatorMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_Main">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Top">
     <item>
      <widget class="QLabel" name="labelPlaylistName">
       <property name="text">
        <string>Playlist Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="playlistNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="fileListWidget">
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_Buttons">
     <item>
      <widget class="QPushButton" name="addFilesButton">
       <property name="text">
        <string>Add Files...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeSelectedButton">
       <property name="text">
        <string>Remove Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_Buttons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="moveFilesCheckBox">
     <property name="text">
      <string>Move files to a new folder (named like playlist)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="hiddenFolderCheckBox">
     <property name="text">
      <string>Make new folder hidden/invisible (dot-prefix/hidden attribute)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="warningLabel">
     <property name="text">
      <string>Warning: Moving files will permanently relocate them from their original location.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string>color: orange;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
</ui>
